var/spack/repos/builtin/packages/perl-star-fusion/package.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from glob import glob
+
+
+class PerlStarFusion(Package):
+    """STAR-Fusion is a component of the Trinity Cancer Transcriptome Analysis
+    Toolkit (CTAT). STAR-Fusion uses the STAR aligner to identify candidate
+    fusion transcripts supported by Illumina reads. STAR-Fusion further
+    processes the output generated by the STAR aligner to map junction reads
+    and spanning reads to a reference annotation set."""
+
+    homepage = "https://github.com/STAR-Fusion/STAR-Fusion"
+    url      = "https://github.com/STAR-Fusion/STAR-Fusion/archive/master.zip"
+
+    version('master', '0d8b79e9fb11a131098870278e84859d')
+
+    depends_on('star', type=('build', 'run'))
+    depends_on('perl-intervaltree', type=('build', 'run'))
+    depends_on('perl-dbi', type=('build', 'run'))
+    depends_on('perl-dbfile', type=('build', 'run'))
+    depends_on('perl-uri-escape', type=('build', 'run'))
+
+    def setup_environment(self, spack_env, run_env):
+        run_env.prepend_path('PATH', join_path(self.prefix, 'util'))
+        run_env.prepend_path('PERL5LIB', join_path(self.prefix, 'lib'))
```

Comment:
If you're going to control where things get installed, it might be better to use the standard directories. Install binaries to `prefix.bin` and install Perl things to `join_path(d.prefix.lib, 'perl5')`.